pom.xml

```diff
@@ -43,6 +43,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+        <!-- bump version when performing releases -->
+        <geomesa.release.version>1.1.0-rc.6</geomesa.release.version>
```

Comment:
My understanding of that plugin was that it still depends on user-input when incrementing the version number.